### ISSUE TITLE
feature requests

### ISSUE THREAD

**Issue by boumboum** [future-architect/cheetah-grid#84]
This rather a feature request as an issue.
Is there any possibility to:

- have column footer (for sums,count, avg, etc)
- have nested tables (a kind of master/detail)
- have grouping of row data

**Comment by ota-meshi** [comment]
@boumboum Thank you for the feature requests.

I think that all feature are awesome!

We will consider adding features.
However, I think that it will take time because it is a difficult.

**Comment by boumboum** [comment]
@ota: I agree, some of those requests are difficult, but IMHO the column footers should be feasible (basically the same as the header but fixed at the bottom of the canvas. The "only" task is to do the (re)calculation based on the datasource or filterdatasource

**Comment by cwwjsyc** [comment]
Need a input Ation like  autocomplete input support.
